shell/common/engine.cc

```diff
@@ -375,12 +375,59 @@ void Engine::HandleSettingsPlatformMessage(PlatformMessage* message) {
   }
 }
 
-void Engine::DispatchPointerDataPacket(const PointerDataPacket& packet,
-                                       uint64_t trace_flow_id) {
+void Engine::DispatchPointerDataPacket(
+    std::unique_ptr<PointerDataPacket> packet,
+    uint64_t trace_flow_id) {
   TRACE_EVENT0("flutter", "Engine::DispatchPointerDataPacket");
   TRACE_FLOW_STEP("flutter", "PointerEvent", trace_flow_id);
+#ifdef ENABLE_IRREGULAR_INPUT_DELIVERY_FIX
```

Comment:
Please perform this sanitization on the platform task runner and in the platform specific subclass for the entrypoint of these events into the engine. In this case, it would be `PlatformViewIOS`. We don't apply platform specific hacks in the "common" classes. This will also get rid of the the `ENABLE_IRREGULAR_INPUT_DELIVERY_FIX`.

Reply:
Done.